xptifw/src/xpti_trace_framework.cpp

```diff
@@ -959,9 +959,10 @@ class Framework {
       // have 'nullptr' for both the Parent and Object only if UserData is
       // provided and the trace_point_type is function_begin/function_end.
       // This allows us to trace function calls without too much effort.
-      std::array<trace_point_type_t, 13> AllowedTypes = {
+      std::array<trace_point_type_t, 14> AllowedTypes = {
           trace_point_type_t::function_begin,
           trace_point_type_t::function_end,
+          trace_point_type_t::diagnostics,
```

Comment:
This will break ABI compatibility. Adding it to the end as the last element may keep compatibility.

Reply:
will update, thanks

Reply:
fixed